databricks-catalog-external-location/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "tags" {
+  description = "Tags to apply to resources"
+  type        = map(string)
+}
+
+variable "bucket_name" {
+  description = "Name of the bucket"
+  type        = string
+}
+
+variable "databricks_external_location_iam_role" {
+  description = "Name of the role for instance profiles"
+  type        = string
+}
+
+variable "name_prefix" {
+  description = "Prefix to apply to resources"
+  type        = string
+}
+
+variable "group_names" {
```

Comment:
Let's change this to `all_privileges_grantees` or equivalent, given the grants for these groups